forge-ai/src/main/java/forge/ai/PlayerControllerAi.java

```diff
@@ -106,6 +106,38 @@ public List<PaperCard> sideboard(Deck deck, GameType gameType, String message) {
             return null;
         }
 
+        Map<PaperCard, PaperCard> sideboardPlan = Maps.newHashMap();
+        List<PaperCard> main = deck.get(DeckSection.Main).toFlatList();
+        List<PaperCard> sideboard = deck.get(DeckSection.Sideboard).toFlatList();
```

Comment:
need check if sideboard even exists:
```
Game-0 > java.lang.NullPointerException: Cannot invoke "forge.deck.CardPool.toFlatList()" because the return value of "forge.deck.Deck.get(forge.deck.DeckSection)" is null
	at forge.ai.PlayerControllerAi.sideboard(PlayerControllerAi.java:126)
	at forge.game.Match.prepareAllZones(Match.java:275)
```